src/components/PollData.js

```diff
@@ -1,21 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
 import {
     FlatList,
     StyleSheet,
     Text,
     View,
     Dimensions,
-    Alert
+    Alert,
+    TouchableOpacity
 } from "react-native";
-import { TouchableOpacity } from "react-native-gesture-handler";
 import AntDesign from 'react-native-vector-icons/AntDesign';
 import { requestRemovePoll } from '../Redux/Action/action';
+import { requestEditTitle } from '../Redux/Action/action';
 import { connect } from "react-redux";
+import { Colors } from "./Colors";
+import UpdatePollTitle from './UpdatePollTitle';
 
 
 
 const PollData = (props) => {
 
+    const [modalValue, setModalValue] = useState(false);
+    const [editTitle, setEditTitle] = useState('')
+
+    const showModal = () => {
+        setModalValue(!modalValue);
+    }
+
+
+    let temp = "";
+    let temp1 = "";
+
+    if (editTitle.length > 0) {
+        const { title, _id } = editTitle[0]
+        temp = title,
+            temp1 = _id
+    }
```

Comment:
what is the use of this block?

Reply:
With the help of this we can get the existing title and id of the poll